compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -3470,6 +3472,28 @@ private STNode parseFuncCall(STNode identifier) {
         return STNodeFactory.createFunctionCallExpressionNode(identifier, openParen, args, closeParen);
     }
 
+    /**
+     * Parse error constructor expression.
+     * <code>functional-constructor-expr := functionally-constructible-type-reference ( arg-list )
+     * functionally-constructible-type-reference := error | type-reference</code>
```

Comment:
Need to add proper line-breaks/paragraph-starts, to render properly. Check other doc comments.

Reply:
Added